WcaOnRails/lib/tasks/db.rake

```diff
@@ -63,7 +63,7 @@ namespace :db do
   namespace :load do
     desc 'Download and import the publicly accessible database dump from the production server'
     task development: :environment do
-      if ENVied.WCA_LIVE_SITE
+      if EnvVars.WCA_LIVE_SITE
```

Comment:
Does this actually work? Given `env_vars`'s behavior I would guess you need to use:
```suggestion
      if EnvVars.WCA_LIVE_SITE?
```
(there are a bunch of these, I'll just suggest on other occurences)